docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -28,23 +28,64 @@ videoBanner: UFhT52d5bYg
 - (!docs/pages/includes/database-access/aws-auto-discovery-prerequisite.mdx!)
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Create a Teleport user
+## Step 1/6. Create a Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 2/5. Create a Database Service configuration
+## Step 2/6. Create a Database Service configuration
 
 (!docs/pages/includes/database-access/token.mdx!)
 
 (!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
 (!docs/pages/includes/database-access/aws-create-db-configuration.mdx type="Redshift" matcher="redshift" protocol="postgres" example="redshift-cluster-1.abcdefghijklm.us-west-1.redshift.amazonaws.com:5438"!)
 
-## Step 3/5. Create an IAM policy for Teleport
+## Step 3/6. Create an IAM Role for user access (optional) 
 
-Teleport needs AWS IAM permissions to be able to:
+Redshift supports two methods of IAM authentication, both of which Teleport
+also supports.
+
+The first method involves accessing Redshift as an existing database user. If
+you are using this traditional IAM authentication, you can skip this step.
```

Comment:
I would clarify "this traditional", which sounds like it could be up for interpretation.

Reply:
Thanks a lot Paul! I was thinking of differentiating them like old way vs new way, but it doesn't feel working very well. 

I've made a new change to just make these two paragraphs more descriptive. What do you think? 9783e8448fccdbe80b1a6fd2fad8a3121e10cbd0